plugins/GSdx/GSTextureCache.cpp

```diff
@@ -48,7 +48,9 @@ GSTextureCache::GSTextureCache(GSRenderer* r)
 
 	m_paltex = theApp.GetConfigB("paltex");
 	m_can_convert_depth &= s_IS_OPENGL; // only supported by openGL so far
-	m_crc_hack_level = theApp.GetConfigI("crc_hack_level");
+	m_crc_hack_level = theApp.GetConfigT<CRCHackLevel>("crc_hack_level");
+	if (m_crc_hack_level == CRCHackLevel::Automatic)
+		m_crc_hack_level = AutoSelectCRCHackLevel(theApp.GetConfigT<GSRendererType>("Renderer"));
```

Comment:
As the source of the problem is the same for the gsrenderer I would solve both in this pr if that is ok with you. That would solve the s_IS_OPENGL issue as well.